rstudio/rsconnect-python
# deploy_html

### Description

Better support for deployment of static html files and bundles.

### QA/Testing

User should be able to deploy html content by passing a path (file or directory) to the new `deploy html` command

`rsconnect deploy html -n server index.html extra_file`

`rsconnect deploy html -n server folder`
